lib/ClangImporter/ImportDecl.cpp

```diff
@@ -8879,7 +8879,20 @@ ClangImporter::Implementation::loadAllMembers(Decl *D, uint64_t extra) {
         if (!member)
           continue;
 
-        enumDecl->addMember(member);
+        // If the decl we're looking into is a namespace decl that has already
+        // been added as a member, we avoid adding it again.
+        bool isANamespace = isa<clang::NamespaceDecl>(member->getClangDecl());
```

Comment:
This isn't just an issue with nested namespaces. Here's another test case that hits the same assertion:
```
namespace Space {
struct A { };
struct B { };
}

using Space::A;
using C = Space::B;

func test1(_ _: C) { }
func test2(_ _: C) { }
func test3(_ _: A) { }
```

I think we should drop the `isANamespace` check altogether and make this a bit more robust. I think there are a few other test cases too.